setup.py

```diff
@@ -77,7 +77,7 @@ def get_readme() -> str:
     packages=find_packages("src"),
     package_dir={"": "src"},
     package_data={"seals": ["py.typed"]},
-    install_requires=["gym"],
+    install_requires=["gym", "numpy", "matplotlib"],
```

Comment:
In practice, I think (might be wrong though) that it doesn't matter because `matplotlib` is installed anyways with the scipy package I think, as I only added it because PyCharm raised a warning, but imports were working fine (no errors). 

In principle, I agree that having it as an explicit dependency is undesirable if it's just for diagnostics.